BridgeAppSDK/SBAOnboardingStepController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  SBAOnboardingStepController.swift
+//  BridgeAppSDK
+//
+//  Copyright © 2016-2017 Sage Bionetworks. All rights reserved.
```

Comment:
Should this just say 2017? (or was this file modified and renamed from an existing one? if so, github doesn't say)